src/sentry/models/actor.py

```diff
@@ -1,10 +1,24 @@
+from collections import defaultdict, namedtuple
+from rest_framework import serializers
```

Comment:
Sorry to nitpick, but this sorts with django, not the core python libraries. IE 3rd party libraries.

Reply:
Oops. Will fix.